### ISSUE TITLE
Error log when integrating via SwiftPM

### ISSUE THREAD

**Issue by mbanasiewicz** [appnexus/mobile-sdk-ios#68]
Thanks for adding support for SwiftPM, everything works with the exception of this error log. 

```
APPNEXUS ERROR  NSString * _Nullable ANPathForANResource(NSString * _Nullable __strong, NSString * _Nullable __strong) -- Could not find resource SDK-Info.plist. Please make sure that all the resources in sdk/resources are included in your app target's "Copy Bundle Resources".
```

I think it's caused by this line `NSDictionary *sdkPlist = [NSDictionary dictionaryWithContentsOfFile:ANPathForANResource(@"SDK-Info", @"plist")];`

<img width="1298" alt="Screenshot 2021-07-16 at 09 33 42" src="https://user-images.githubusercontent.com/1093407/125910048-ce7216d9-f7ce-47aa-a128-5424aea4fab0.png">

We can see that the requests have proper sdk version.

Is this something we should worry about?

**Comment by Kowshickkarthick** [comment]
Hi @mbanasiewicz ,

We will look in to this error msg….. and try providing a fix in our next release.
You can safely ignore this Error message for now since it does not affect any functionality in the SDK.

The error case is already checked and handled in code.
`return AN_SDK_VERSION;` will be returning the actual SDK version if we are not able to fetch it from SDK-Info.plist file. Thats the reason why you are able to see SDK version in ad request.

**Comment by asharmaa** [comment]
SDK v7.16.0+ has fix for issue #68.
